tests/test_modeling_common.py

```diff
@@ -971,3 +974,70 @@ def test_top_k_top_p_filtering(self):
 
         self.assertTrue(torch.allclose(non_inf_expected_output, non_inf_output, atol=1e-12))
         self.assertTrue(torch.all(torch.eq(non_inf_expected_idx, non_inf_idx)))
+
+    @require_torch
+    def test_postprocess_next_token_scores(self):
+
+        config = MarianConfig.from_pretrained("Helsinki-NLP/opus-mt-fr-en")
+        model = MarianMTModel(config=config)
```

Comment:
you might want to test a smaller model here.

Reply:
you should definitely test a tiny model since the tiny model doesn't matter.
use `sshleifer/tiny-marian-en-de/`